client/lobby/SelectionTab.cpp

```diff
@@ -639,12 +751,13 @@ void SelectionTab::parseCampaigns(const std::unordered_set<ResourceID> & files)
 	allItems.reserve(files.size());
 	for(auto & file : files)
 	{
-		auto info = std::make_shared<CMapInfo>();
+		auto info = std::make_shared<ElementInfo>();
 		//allItems[i].date = std::asctime(std::localtime(&files[i].date));
 		info->fileURI = file.getName();
 		info->campaignInit();
-		if(info->campaign)
+		if(info->campaign) {
 			allItems.push_back(info);
+		}
```

Comment:
1. We place opening brackets on new line
2. No need for brackets for simple, single-line statements
```suggestion
		if(info->campaign)
			allItems.push_back(info);
```